uui/components/overlays/DropdownContainer.tsx

```diff
@@ -11,7 +11,10 @@ export interface DropdownContainerProps extends uuiDropdownContainerProps, Dropd
 
 function applyDropdownContainerMods(mods: DropdownContainerMods) {
     return [
-        css.root, mods.vPadding && `vPadding-${mods.vPadding}`, mods.padding && `padding-${mods.padding}`,
+        'uui-dropdown_container',
```

Comment:
may be we need to move this global class into uui-components

Reply:
We agreed to add to all components global classes according component name, in uui-components 'uui-dropdown-body' doesn't connected to this approach